Project.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+using namespace std;
+
+class User{
+    public:
+    string Username;
+    string Email;
+    User(string username,string email){
+        Username = username;
+        Email = email;
+    }
+    void print(){
+        std::cout<<"Username - "<<Username<<endl;
```

Comment:
Replace 'endl' with '\n' for better performance in output operations, as 'endl' forces a flush of the output buffer. [medium]